doc/releases/quincy.rst

```diff
@@ -0,0 +1,3656 @@
+* Integration with new NFS mgr module
+* Ability to zap osds as they are removed
+* cephadm agent for increased performance/scalability
+
+Dashboard
+~~~~~~~~~
+* Day 1: the new "Cluster Expansion Wizard" will guide users through post-install steps:
+  adding new hosts, storage devices or services.
+* NFS: the Dashboard now allows users to fully manage all NFS exports from a single place.
+* New mgr module (feedback): users can quickly report Ceph tracker issues
+  or suggestions directly from the Dashboard or the CLI.
+* New "Message of the Day": cluster admins can publish a custom message in a banner.
+* Cephadm integration improvements:
+   * Host management: maintenance, specs and labelling,
+   * Service management: edit and display logs,
+   * Daemon management (start, stop, restart, reload),
+   * New services supported: ingress (HAProxy) and SNMP-gateway.
+* Monitoring and alerting:
+   * 43 new alerts have been added (totalling 68) improving observability of events affecting:
+     cluster health, monitors, storage devices, PGs and CephFS.
+   * Alerts can now be sent externally as SNMP traps via the new SNMP gateway service
+     (the MIB is provided).
+   * Improved integrated full/nearfull event notifications.
+   * Grafana Dashboards now use grafonnet format (though they're still available
+     in JSON format).
+   * Stack update: images for monitoring containers have been updated.
+     Grafana 8.3.5, Prometheus 2.33.4, Alertmanager 0.23.0 and Node Exporter 1.3.1.
+     This reduced exposure to several Grafana vulnerabilities (CVE-2021-43798,
+     CVE-2021-39226, CVE-2021-43798,  CVE-2020-29510, CVE-2020-29511).
+
+RADOS
+~~~~~
+
+* OSD: Ceph now uses `mclock_scheduler` for BlueStore OSDs as its default
+  `osd_op_queue` to provide QoS. The 'mclock_scheduler' is not supported
+  for Filestore OSDs. Therefore, the default 'osd_op_queue' is set to `wpq`
+  for Filestore OSDs and is enforced even if the user attempts to change it.
+  For more details on configuring mclock see,
+
+  https://docs.ceph.com/en/quincy/rados/configuration/mclock-config-ref/
+
+  An outstanding issue exists during runtime where the mclock config options
+  related to reservation, weight and limit cannot be modified after switching
+  to the `custom` mclock profile using the `ceph config set ...` command.
+  This is tracked by: https://tracker.ceph.com/issues/55153. Until the issue
+  is fixed, users are advised to avoid using the 'custom' profile or use the
+  workaround mentioned in the tracker.
+
+* MGR: The pg_autoscaler can now be turned `on` and `off` globally
+  with the `noautoscale` flag. By default, it is set to `on`, but this flag
+  can come in handy to prevent rebalancing triggered by autoscaling, during
+  cluster upgrade and maintenance. Pools can now be created with the `--bulk`
+  flag, which allows the autoscaler to allocate more PGs to such pools. This
+  can be useful to get better out of the box performance for data-heavy pools.
```

Comment:
Should we recommend users to set the autoscaler to warn before a major upgrade, to prevent surprises?

Reply:
I suggest removing the comma in `autoscaling, during`, and agree with Dan -- along with a reminder to re-enable when the cluster is stable.

Reply:
done, and mentioned the noautoscale flag in the upgrade notes